docs/getting-started/create-defend-policy-api.asciidoc

```diff
@@ -0,0 +1,813 @@
+[[create-defend-policy-api]]
+= Create an {elastic-defend} policy using API
+
+:frontmatter-description: Create and customize an {elastic-defend} policy through the API.
+:frontmatter-tags-products: [security, defend]
+:frontmatter-tags-content-type: [how-to]
+:frontmatter-tags-user-goals: [manage]
+
+In addition to <<configure-endpoint-integration-policy, configuring an {elastic-defend} policy>> through the {elastic-sec} UI, you can create and customize an {elastic-defend} policy through the API. This is a three-step process involving the {fleet} API. For detailed information on {fleet} API, refer to {fleet-guide}/fleet-api-docs.html[{fleet} API documentation].
+
+[discrete]
+[[create-agent-policy]]
+== Step 1: Create an agent policy
+
+Make a call to create an agent policy that you will add your {elastic-defend} integration to. Replace `<KIBANA-VERSION>` with your version of {kib}.
```

Comment:
```suggestion
Use the following API call to create a new agent policy where you will add your {elastic-defend} integration. Replace `<KIBANA-VERSION>` with your version of {kib}.
```
Maybe worth specifying "API call" the first few times "call" appears in this doc